spec/support/trusted.rb

```diff
@@ -0,0 +1,4 @@
+RSpec.configure do |c|
+  c.trusted_node_data = true
+  c.trusted_server_facts = true
```

Comment:
Surprisingly https://github.com/puppetlabs/rspec-puppet#trusted_node_data and https://github.com/puppetlabs/rspec-puppet#trusted_server_facts are off by default.